tee-worker/core/direct-rpc-server/src/rpc_ws_handler.rs

```diff
@@ -20,8 +20,8 @@ use crate::sgx_reexport_prelude::*;
 
 use crate::{DetermineWatch, RpcConnectionRegistry, RpcHash};
 use itc_tls_websocket_server::{error::WebSocketResult, ConnectionToken, WebSocketMessageHandler};
+use itp_utils::debug as lit_debug;
```

Comment:
can we just use `debug`, without prefix `lit_` ? Will it introduce any conflict?
If no conflict, I would suggest only `debug` is sufficient. 
so we will only change: 
`use log::*` to `use itp_utils::*`

We may need to take care of `sgx` vs `std`.

Reply:
Yes we can but I prefer different names from `log` crate macros.
What do you think about renaming our macros, example `debug` -> `lit_debug` ?

Reply:
I actually prefer renaming the macros to `lit_` prefixed, to clearly differ from the known log macros.

And it seems we are using alias anyway in most of the cases

Reply:
@kziemianek let's renaming our macros directly to `lit_***`. And then merge this PR.

Reply:
I'll remove them as they seems to be redundant since we locked log level for production build at `INFO`.